deployments/container/nvidia-mig-manager-example-hopper.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       serviceAccountName: nvidia-mig-manager-service-account
       containers:
       - name: nvidia-mig-manager
-        image: nvcr.io/nvidia/cloud-native/k8s-mig-manager:v0.6.0-ubi8
+        image: nvcr.io/nvidia/cloud-native/k8s-mig-manager:v0.7.0-ubi8
```

Comment:
Not a blocking comment. But we should consider publishing ubi9 images at some point

Reply:
Do you mean that we need to replace the images? 

Note that since we copy the `mig-parted` executable from the container to the host, we want to ensure that we're using as low a base version as possible to avoid issues with GLIBC dependencies.

Reply:
I meant use `ubi9` as the base image instead of `ubi8`